src/ShaderTools.Editor.VisualStudio/source.extension.vsixmanifest

```diff
@@ -14,9 +14,10 @@
   <Installation InstalledByMsi="false">
     <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="14.0" />
     <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="15.0" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="16.0" />
   </Installation>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0]" DisplayName="Visual Studio core editor" />
```

Comment:
I think this should be `[15.0,17.0)` so that it works with all point-releases of VS2019.

Reply:
Yes, fixed it!

Reply:
> I think this should be `[15.0,17.0)` so that it works with all point-releases of VS2019.

Yes, fixed it!